pkgs/top-level/python-packages.nix

```diff
@@ -2616,17 +2623,29 @@ in modules // {
       py.test $out/${python.sitePackages}
     '';
 
-    # Marked broken since according to test
+    # FIXME: make gdk_pixbuf dependency optional (as wel as xcfffi)
     # Happens with 0.7.1 and 0.7.2
     # OSError: dlopen() failed to load a library: gdk_pixbuf-2.0 / gdk_pixbuf-2.0-0
 
-    patchPhase = ''
+    patches = [
```

Comment:
On Sun, Feb 07, 2016 at 05:32:44AM -0800, Nikolay Amiantov wrote:

> >      # Happens with 0.7.1 and 0.7.2
> >      # OSError: dlopen() failed to load a library: gdk_pixbuf-2.0 / gdk_pixbuf-2.0-0
> >  
> > -    patchPhase = ''
> > +    patches = [
> 
> Usual way before new Nix release was to use _some_ checksum and allow it to fail (it prints actual checksum in an error message). With Nix 1.11 we now have `nix-prefetch-url -A pythonPackages.cairocffi.src`, it _may_ work with `fetchpatch` but I haven't tried.
> 

0gb570z3ivf1b0ixsk526n3h29m8c5rhjsiyam7rr3x80dp65cdl

Will check it on other machines, then poke you to check and merge.